src/app/page.tsx

```
@@ -26,10 +26,55 @@ export default function Dashboard() {
-  const [isSeeded, setIsSeeded] = useState(false)
+  const [isSeeded, setIsSeeded] = useState(true) // Default to true to avoid showing prompt initially
+  const [seedingInProgress, setSeedingInProgress] = useState(false)
 
-  const seedDatabase = async () => {
+  const checkAndAutoSeed = async () => {
     try {
+      setSeedingInProgress(true)
+
+      // Check if database needs seeding
+      const statusResponse = await fetch('/api/seed/status')
+      const statusData = await statusResponse.json()
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)